source/javascripts/components/ProductTour/ProductTooltip.tsx

```diff
@@ -38,17 +40,20 @@ export const ProductTooltip = ({
 
 	return (
 		<Box
-			left={rect?.x}
-			top={(rect?.y ?? 0) + (rect?.height ?? 0) + 20}
+			left={tip.position === "right" ? rect!.x + rect!.width + 37: rect!.width/2 - 210}
+			top={tip.position === "right" ? rect!.bottom/2  - 22: (rect?.y ?? 0) + (rect?.height ?? 0) + 40}
 			className="product-tooltip"
+			position="relative"
 		>
 			<Box display="flex" flexGrow="1">
-				<div className="arrow-up" style={{ left: rect!.width / 2 - 5 }} />
+				{ tip.position === "right" ? 
+					<div className="arrow-left" /> : 
```

Comment:
kind of a small thing. wonder if its easier to read if the conditional is for the class name instead of changing the element like:
```tsx
<div className={`arrow-${tip.position === "right" ? "left" : "up"}`} />
```

Reply:
Good catch! This has been updated.